src/libraries/System.Private.CoreLib/src/System/Number.Formatting.cs

```diff
@@ -272,7 +272,7 @@ internal static partial class Number
 
         // Optimizations using "TwoDigits" inspired by:
         // https://engineering.fb.com/2013/03/15/developer-tools/three-optimization-tips-for-c/
-        private static ReadOnlySpan<byte> TwoDigitsCharsAsBytes =>
+        private static readonly byte[] TwoDigitsCharsAsBytes =
```

Comment:
Would it make sense to place the Mono workaround under `#if MONO`? It is just a few places.

Reply:
Is this PR introducing a regression for NativeAOT then? If I am reading the code correctly, code that uses TwoDigitsCharsAsBytes is going to have extra cctor trigger with this PR.

Reply:
even 
```cs
static readonly char[] TwoDigitsCharsAsBytes = "90919293949596979899".ToArray();
```
is not preinitialized today